cypress/integration/keyword.test.js

```diff
@@ -0,0 +1,26 @@
+// <reference types="cypress" />
+
+describe('Keyword search', () => {
+  const keyword = 'Terrier';
+  const resultCount = 4;
```

Comment:
@russom-woldezghi I retested this yesterday after destroying my vagrant and provisioning it again. I now get 2 results consistently on all sites including this one, so I think 2 might be correct. Can you verify with the current demo environment?

Reply:
My result matches this